test/fixtures/dialects/postgres/postgres_create_trigger.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TRIGGER foo AFTER INSERT ON bar EXECUTE FUNCTION proc(args);
+
+CREATE TRIGGER foo BEFORE INSERT on bar EXECUTE FUNCTION proc(args);
+
+CREATE TRIGGER foo AFTER UPDATE OF bar, baz ON bar EXECUTE FUNCTION proc(args);
+
+CREATE TRIGGER foo INSTEAD OF DELETE ON bar FROM baz DEFERRABLE INITIALLY DEFERRED FOR EACH STATEMENT EXECUTE FUNCTION proc(args);
+
+CREATE TRIGGER foo BEFORE INSERT ON bar WHEN (a=b) EXECUTE FUNCTION proc(args);
+
+CREATE OR REPLACE CONSTRAINT TRIGGER foo BEFORE INSERT ON bar EXECUTE FUNCTION proc(args);
+
+CREATE TRIGGER foo BEFORE INSERT ON bar REFERENCING OLD TABLE as old_table NEW TABLE AS new_table EXECUTE PROCEDURE proc(args);
```

Comment:
Could you include the examples from https://www.postgresql.org/docs/14/sql-createtrigger.html?

And similarly for the other tests.

Always good to include the examples when they're good enough to give.

Reply:
Sorted - bar one. Fixed another issue getting one passing, adding another issue for the other. Can't getr `table.*` to parse

Reply:
Want to merge as is and come back to this? Or still working on it?

Reply:
I've added an issue - let's merge as is. I'll come back to this once 0.8 is out the door